src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -0,0 +1,175 @@
+{# wf_featured_snippet: Learn best practices for good user experiences with the new autoplay policies in Chrome. #}
+{# wf_blink_components: Blink>Media #}
+
+# Autoplay Policy Changes {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome's autoplay policies are about to change in 2018 and I'm here to tell
+you why and how this is going to affect video playback with sound. Spoiler
+alert: Users are going to love it!
+
+<figure>
+  <a href="https://imgflip.com/i/ngd6c">
+    <img height="241px" src="https://i.imgflip.com/ngd6c.jpg"
+         title="I will find you and I will pause you"/>
+  </a>
+  <a href="https://imgur.com/9pj5LGN">
+    <img height="241px" src="http://i.imgur.com/9pj5LGN.jpg"
+         title="I don't remember asking you to autoplay a goddamn thing"/>
+  </a>
+  <a href="https://imgur.com/a/p1ZjC">
+    <img height="241px" src="https://i.imgur.com/aFO9wAml.jpg"
+         title="One does not simply autoplay videos"/>
+  </a>
+  <figcaption>
+    <b>Figure 1.</b>
+    Internet memes tagged "autoplay" found on
+    <a href="https://imgflip.com">Imgflip</a> and
+    <a href="https://imgur.com">Imgur</a>
+  </figcaption>
+</figure>
+
+## New Behaviors {: #new-behaviors }
+
+As you may have [noticed], web browsers are moving towards stricter autoplay
+policies in order to improve the web experience for users, minimize the
+incentives to install extensions that block ads, and reduce data consumption on
+expensive and/or constrained network.
+
+With these new autoplay policies, the Chrome team aims to provide a greater
+control to users over content playing in their browser. It will also benefit
+publishers who have legitimate autoplay use cases.
+
+Chrome's autoplay policies are simple:
+
+- Muted autoplay is always allowed.
+- Autoplay with sound is allowed if any of the following conditions are met:
+    - User has interacted with the site (click, tap, etc.)
+    - [Media Engagement Index](#mei) threshold is crossed (desktop only)
+    - Site has been installed using the ["Add to Homescreen" flow] (mobile only)
+- Top frame can [delegate autoplay permission](#iframe) to their iframes to
+  allow autoplay with sound.
+
+### Media Engagement Index (MEI) {: #mei }
+
+The MEI measures an individual's propensity to consume media on a site.
+Chrome's current approach is a ratio of visits to significant media playback
+events per origin:
+
+- Consumption of the media (audio/video) must be greater than 7 seconds.
+- Audio must be present and unmuted.
+- Tab with video is active.
+- Size of the video (in px) must be greater than [200x140].
+
+From that, Chrome calculates a Media Engagement score which is highest on sites
+where media is played on a regular basis. When it is high enough, media playback
+is allowed to autoplay on desktop only.
+
+Note that you can check out user's MEI at the <i>chrome://media-engagement</i>
+internal page in Chrome 62.
```

Comment:
Presumably, the only MEI I can view this way is my own. This phrasing makes it sound like I can view any arbitrary user's MEI. Please try this sentence again.

Reply:
Tried again. Thanks!